host/frontend/liboperator/operator/operator.go

```diff
@@ -97,9 +99,45 @@ func CreateHttpHandlers(
 	router.HandleFunc("/infra_config", func(w http.ResponseWriter, r *http.Request) {
 		ReplyJSONOK(w, config)
 	}).Methods("GET")
+	remoteUrl, proxy, err := getOpenWrtProxy()
+	if err == nil {
+		router.HandleFunc("/openwrt", forwardWithoutUrl(remoteUrl, proxy))
+		// We cannot set up the base url for luci, so just add necessary paths as well.
+		router.HandleFunc("/cgi-bin/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+		router.HandleFunc("/luci-static/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+	}
 	return router
 }
 
+func getOpenWrtProxy() (*url.URL, *httputil.ReverseProxy, error) {
+	luciUrl := "http://192.168.96.2"
```

Comment:
The ip address should be in a constant or, even better, an argument to the function

Reply:
made it constant, and added an argument as well.